contracts/src/v0.8/ccip/test/NonceManager.t.sol

```diff
@@ -128,7 +137,27 @@ contract NonceManager_applyPreviousRampsUpdates is OnRampSetup {
     );
   }
 
-  function test_ZeroInput() public {
+  function test_PreviousRampAlreadySet_overrideAllowed_success() public {
+    NonceManager.PreviousRampsArgs[] memory previousRamps = new NonceManager.PreviousRampsArgs[](1);
+    address prevOffRamp = makeAddr("prevOffRamp");
+    previousRamps[0] = NonceManager.PreviousRampsArgs({
+      remoteChainSelector: DEST_CHAIN_SELECTOR,
+      prevRamps: NonceManager.PreviousRamps(address(0), prevOffRamp),
+      overrideExistingRamps: true
+    });
+
+    s_outboundNonceManager.applyPreviousRampsUpdates(previousRamps);
+
+    previousRamps[0] = NonceManager.PreviousRampsArgs({
```

Comment:
nit: I think this line is not needed, since it's setting the same data as the `previousRamps[0]` above